cmd/cerbosctl/list/list.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 Zenauth Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package list
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	policy "github.com/cerbos/cerbos/api/genpb/cerbos/policy/v1"
+	"github.com/cerbos/cerbos/client"
+	"github.com/cerbos/cerbos/cmd/cerbosctl/internal"
+	"github.com/cerbos/cerbos/internal/util"
+)
+
+var listPoliciesFlags = internal.NewListPoliciesFilterDef()
+
+func NewListCmd(fn internal.WithClient) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List active policies",
+		RunE:  fn(runListCmdF),
+	}
+
+	cmd.Flags().AddFlagSet(listPoliciesFlags.FlagSet())
+
+	return cmd
+}
+
+func runListCmdF(c client.AdminClient, cmd *cobra.Command, _ []string) error {
+	opts, err := internal.GenListPoliciesFilterOptions(listPoliciesFlags)
+	if err != nil {
+		return fmt.Errorf("error generating list options: %w", err)
+	}
+
+	policies, err := c.ListPolicies(context.Background(), opts...)
+	if err != nil {
+		return fmt.Errorf("error while requesting policy list: %w", err)
+	}
+
+	if err = printPolicies(cmd.OutOrStdout(), policies, listPoliciesFlags.OutputFormat()); err != nil {
+		return fmt.Errorf("could not print policies: %w", err)
+	}
+
+	return nil
+}
+
+func printPolicies(w io.Writer, policies []*policy.Policy, format string) error {
+	switch format {
+	case "json":
+		for _, policy := range policies {
+			b, err := protojson.Marshal(policy)
+			if err != nil {
+				return fmt.Errorf("could not marshal policy: %w", err)
+			}
+			fmt.Fprintf(w, "%s\n", b)
+		}
+	case "yaml":
+		for _, policy := range policies {
+			err := util.WriteYAML(w, policy)
+			if err != nil {
+				return fmt.Errorf("could not write policy: %w", err)
+			}
+			fmt.Fprintln(w, "---")
+		}
+	default:
+		headerFmt := color.New(color.FgGreen, color.Underline).SprintfFunc()
+
+		tbl := table.New("NAME", "KIND", "DEPENDENCIES")
```

Comment:
I think VERSION is required here. This is the output when there are two versions of `leave_request` in the server. 

```
NAME              KIND           DEPENDENCIES      
my_derived_roles  DERIVED_ROLES  -                 
donald_duck       PRINCIPAL      -                 
leave_request     RESOURCE       my_derived_roles  
leave_request     RESOURCE                         
```

Ideally the output should be sorted by name as well.

Reply:
Makes sense, going to add it 👍 And yeah this can be sorted alphabetically.